packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -738,6 +759,13 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         }
     }
 
+    // Verify not tombstoned is seperate from not closed as the setConnectionState method is internal
+    private verifyNotTombstoned() {
```

Comment:
What pros/cons do you see to just putting this check inside `verifyNotClosed`?  I'm concerned that later someone would call `verifyNotClosed` but neglect to call this.

Reply:
The problem of putting it inside `verifyNotClosed` is that the runtime will fail because `verifyNotClosed` is called in `setConnectionState`. Every time the connection state for the runtime changes for a container, the connection state for a datastore would change and if it's tombstoned, we'll throw an error. So `verifyNotTombstoned` was written.